ArmNavigation/new servo.py

```
@@ -0,0 +1,33 @@
+#include "Servo.h"
+
+// Motors
```

Ruff

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

Statement ends with an unnecessary semicolon

(E703)